axlearn/common/flash_attention/tpu_attention_test.py

```diff
@@ -97,12 +92,11 @@ def test_to_splash_mask(self, mask, expected):
 
     @parameterized.product(
         batch_size=[4],
-        seq_len=[1024, 32768],
+        seq_len=[1024],
```

Comment:
Since the sliding window size is 1024, it will be useful to keep a test case for seq_len > 1024. We can enable the test only on TPU if it's too slow on CPU. We can also use a seq_len such as 2048 for cpu if it's fast enough.

Reply:
Done. I changed it back to match the first PR.
![image](https://github.com/user-attachments/assets/c39ea224-0c3e-41b8-9439-e9e02a192cb4)